absorption_modalaSean.py

```diff
@@ -288,14 +264,16 @@ def smooth(norm_flux, box_size):
             count2=0# this is so b/c if the code encounters an other absorption feature which is wider than 600km/s, the code is going to go through the if statement on line 205
             EW_ind=[]
         
-        if jjjs == lst:
+        if value == min_limit_value:
             BI_total= round(sum(BI_mid),2)         
             BI_all.append(BI_total)    
             BI_all_individual.append(BI_individual)
             EW_all_individual.append(EW_individual)
    
     if (len(vmaxs) != 0) or (plotall == 'yes'): 
-        yes=(str(file_index)+';' + '  name: ' + str(spectrum_file_name) + '\n' + 'BI (-30000 > v > -60,000): ' + str(BI_total) + '\n' +  'vmins: ' + str(vmins) + '\n' + 'vmaxs: '+str(vmaxs) + '\n' + 'BI_individual: '+ str(BI_individual) + '\n' + 'EW_individual: '+ str(EW_individual) + '\n' + 'Depth: '+ str(final_depth_individual) +'\n'+'\n')
+        yes=(str(file_count)+';' + str(spectrum_file_name) + '\n' + 'BI (' + str(max_velocity_limit) + ' > v > ' + str(min_velocity_limit) +'): ' + str(BI_total) + '\n' 
```

Comment:
This seems too long, you can make it somewhat shorter by using the f-string instead:
https://www.geeksforgeeks.org/formatted-string-literals-f-strings-python/